README.md

```diff
@@ -106,6 +120,22 @@ aggregators:
     hosts : "agg-31"
     group : l3-agg
 
+samplers:
+  - names       : *sampler-hosts
```

Comment:
I think this would be just "name : samplers". Then maestro automatically applies this one configuration object across all hosts in the 'hosts'  list.